app/_src/policies/meshcircuitbreaker.md

```diff
@@ -0,0 +1,303 @@
+name: web-to-backend-circuit-breaker
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+      default:
+        connectionLimits:
+          maxConnections: 2
+          maxPendingRequests: 8
+          maxRetries: 2
+          maxRequests: 2
+```
+
+We will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/{{ page.version }}/reference/http-api).
+{% endtab %}
+{% endtabs %}
+
+#### Outlier detection for inbound traffic to backend service
+
+{% tabs protocol useUrlFragment=false %}
+{% tab protocol Kubernetes %}
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshCircuitBreaker
+metadata:
+  name: backend-inbound-outlier-detection
+  namespace: {{site.mesh_namespace}}
+spec:
+  targetRef:
+    kind: MeshService
+    name: web
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        outlierDetection:
+          interval: 5s
+          baseEjectionTime: 30s
+          maxEjectionPercent: 20
+          splitExternalAndLocalErrors: true
+          detectors:
+            totalFailures:
+              consecutive: 10
+            gatewayFailures:
+              consecutive: 10
+            localOriginFailures:
+              consecutive: 10
+            successRate:
+              minimumHosts: 5
+              requestVolume: 10
+              standardDeviationFactor: 1.9
+            failurePercentage:
+              requestVolume: 10
+              minimumHosts: 5
+              threshold: 85
+```
+
+We will apply the configuration with `kubectl apply -f [..]`.
+{% endtab %}
+
+{% tab protocol Universal %}
+
+```yaml
+type: MeshCircuitBreaker
+name: backend-inbound-outlier-detection
+mesh: default
+spec:
+  targetRef:
+    kind: MeshService
+    name: web
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        outlierDetection:
+          interval: 5s
+          baseEjectionTime: 30s
+          maxEjectionPercent: 20
+          splitExternalAndLocalErrors: true
+          detectors:
+            totalFailures:
+              consecutive: 10
+            gatewayFailures:
+              consecutive: 10
+            localOriginFailures:
+              consecutive: 10
+            successRate:
+              minimumHosts: 5
+              requestVolume: 10
+              standardDeviationFactor: 1.9
+            failurePercentage:
+              requestVolume: 10
+              minimumHosts: 5
+              threshold: 85
+```
+
+We will apply the configuration with `kumactl apply -f [..]` or via the [HTTP API](/docs/{{ page.version }}/reference/http-api).
+{% endtab %}
+{% endtabs %}
+
+## Connection limits configuration
+
+- **`maxConnections`** - (optional) The maximum number of connections allowed to be made to the upstream cluster. If not
+  specified then equal to "1024".
+- **`maxConnectionPools`** - (optional) The maximum number of connection pools per cluster that are concurrently
```

Comment:
not super clear that it's Envoy cluster. Btw. what does it mean that something has multiple connection pools? I thought that there is one connection pool for destination

Reply:
I'm not sure what you mean by the first sentence, but for the second:
> Each worker thread maintains its own connection pools for each cluster, so if an Envoy has two threads and a cluster with both HTTP/1 and HTTP/2 support, there will be at least 4 connection pools.
> https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/connection_pooling#number-of-connection-pools

Reply:
I think it's addressed, please reopen conversation if you think otherwise